src/index.php

```diff
@@ -35,10 +35,10 @@
 	die();
 }
 
+require_once __DIR__ . '/vendor/autoload.php';
 require_once __DIR__ . '/lib/svgsupport.php';
 
 if ( defined( 'ABSPATH' ) ) {
-	include_once __DIR__ . '/vendor/autoload.php';
```

Comment:
I think this was the root cause. we were only autoloading if `ABSPATH` was defined, which seems silly.

This was to work around test-suite not snapshot-ing the vendor-ing. I'll just be more straightforward.